content/https/https-3/contents.lr

```diff
@@ -0,0 +1,14 @@
+_model: question
+---
+title: What's the difference between Tor and DoH (DNS over HTTPS)?
+---
+seo_slug: difference-between-Tor-DoH
+---
+description:
+DNS over HTTPS (DoH) is a protocol for performing remote DNS resolution via the HTTPS protocol.
+A goal of the method is to increase user privacy and security by preventing eavesdropping and manipulation of DNS data by man-in-the-middle attacks by using the HTTPS protocol to encrypt the data between the DoH client and the DoH-base DNS resolver. 
+Encryption by itself does not protect privacy, encryption is simply a method to obfuscate the data.

```

Comment:
Encryption protects privacy in that it limits visibility of data to designated/intended endpoints. "obfuscate" used here is correct, in the sense the it makes the data obscure to any but the intended parties.